### ISSUE TITLE
Use symfony process helper

### ISSUE THREAD

**Issue by cmuench** [netz98/n98-magerun2#1003]
## Current Implementation

Currently we have own logic in our util section to execute processes.
We should evaluate if it's better to use the build in symfony process helper.

https://symfony.com/doc/4.4/components/console/helpers/processhelper.html

## Expected Benefits
Reduce anount of maintained code.

**Comment by torhoehn** [comment]
@cmuench You mean https://github.com/netz98/n98-magerun2/blob/develop/src/N98/Util/ProcessArguments.php?

**Comment by cmuench** [comment]
@torhoehn No. It's about replacing the manual "exec" calls in the code base with a more standardized way how it's handled in Symfony applications.
We have own util classes and re-invent the wheel.
